pkg/opensearch/response_parser.go

```diff
@@ -27,8 +27,10 @@ const (
 	filtersType     = "filters"
```

Comment:
Random thought for a future pr: I'm wondering if we should break this file down a bit into smaller files? like would this be easier to read maybe if the different response parsers had their own files?